docs/en/binance-loom-ethereum.md

```diff
@@ -0,0 +1,512 @@
+}
+```
+
+The `_load` function from the `UniversalSigning` class does a couple of things:
+
+- first, it loads some values we don't want to hardcode (like the `chainId`) from a JSON file by calling the `_loadNetworkConfiguration` function.
+- sets up a signer and gets your Ethereum address
+- creates a new temporary provider for Loom,
+- checks if a mapping exitsts. If not, it creates one.
+- lastly, it returns the mapping togehter with the temporary provider and the `client`
+
+Here's how the function looks like:
+
+```js
+async _load (web3Ethereum) {
+    this._loadNetworkConfiguration()
+    const client = this._createClient()
+    client.on('error', console.error)
+    const callerAddress = await this._setupSigner(client, web3Ethereum.currentProvider)
+    console.log('callerAddress: ' + callerAddress)
+    const loomProvider = await this._createLoomProvider(client, callerAddress)
+    const web3Loom = new Web3(loomProvider)
+    let accountMapping = await this._loadMapping(callerAddress, client)
+    if (accountMapping === null) {
+      console.log('Create a new mapping')
+      const signer = getMetamaskSigner(web3Ethereum.currentProvider)
+      await this._createNewMapping(signer)
+      accountMapping = await this._loadMapping(callerAddress, client)
+      console.log(accountMapping)
+    } else {
+      console.log('mapping already exists')
+    }
+    console.log('mapping.ethereum: ' + accountMapping.ethereum.toString())
+    console.log('mapping.plasma: ' + accountMapping.plasma.toString())
+    return { web3Loom, accountMapping, client }
+  }
+```
+
+Now, let's get back to our function from the `BinanceExtdevRinkeby` class. Next, it initializes the contracts with `_getContracts`, and calls a function that we'll listen for events- `_filterEvents`. This way, every time our balance changes, the UI will get updated. Lastly, the function reads the initial balance by running `_refreshBalance`.
+
+## Moving tokens from Loom to Ethereum
+
+To move tokens from Loom to Ethereum, we follow a two-step process:
+
+ - first, we approve the transfer gateway to take the fee:
+ 
+```js
+await this.ethCoin.approveFee()
+```
+
+ - once the transfer gateway is approved to take the fee, we run the `withdrawToEthereum` function on the `bep2Coin` object:
+
+```js
+async withdrawToEthereum (amount) {
+  EventBus.$emit('updateStatus', { currentStatus: 'Transferring to Extdev Gateway.' })
+  await this._transferCoinsToExtdevGateway(amount)
+  EventBus.$emit('updateStatus', { currentStatus: 'Getting withdrawal receipt.' })
+  const data = await this._getWithdrawalReceipt()
+  EventBus.$emit('updateStatus', { currentStatus: 'Withdrawing from Rinkeby Gateway.' })
+  await this._withdrawCoinsFromRinkebyGateway(data)
+}
+```
+
+Again, this function is pretty straightforward. First, it transfers the tokens to the transfer gateway contract on the Loom side:
+
+```js
+async _transferCoinsToExtdevGateway (amount) {
+    const amountInt = amount * 100000000
```

Comment:
Use `BN` for any kind of token arithmetic, don't want to teach people bad habits.